src/MissionManager/ComplexMissionItem.cc

```diff
@@ -37,7 +37,11 @@ QStringList ComplexMissionItem::presetNames(void)
 
     settings.beginGroup(presetsSettingsGroup());
     settings.beginGroup(_presetSettingsKey);
-    return settings.childKeys();
+
+    names << "Select a preset";
```

Comment:
I wonder if it would add some clarity if this text was _italic_?

Reply:
Yeah I wasn't quite sure how to do it, I found one way but it was a little bit hacky feeling.